README.md

```diff
@@ -1,70 +1,80 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Stocks Metrics
 
-## Available Scripts
+> A React-Redux stock metrics application.
+Get the best-performing stocks daily see stocks prices, their changes, and the most useful information of each stock.
 
-In the project directory, you can run:
 
-### `npm start`
+![screenshot](./img/mobileVersion.png)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Explanation Video
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+<a href="https://www.loom.com/share/c6db8718465647e5a863f4314c313072">
+    <p>Loom | Free Screen & Video Recording Software - Watch Video</p>
+    <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/c6db8718465647e5a863f4314c313072-with-play.gif">
+</a>
```

Comment:
- [ ] In the video you are expected to highlight a particular piece of code that you find interesting. Kindly record an additional video discussing this.